torchao/quantization/utils.py

```diff
@@ -22,6 +22,13 @@
     "TORCH_VERSION_AFTER_2_3",
 ]
 
+try:
+    import lm_eval  # pyre-ignore[21]  # noqa: F401
+
+    _lm_eval_available = True
+except:
+    _lm_eval_available = False
+
```

Comment:
With python 3.8, importing `_lm_eval_available` is failing. Could we add `_lm_eval_available = False` before the `try-except` block? Thanks!

```
ImportError: cannot import name '_lm_eval_available' from 'torchao.utils'
```